python/sauron-core/core/analysis/classifier.py

```
@@ -0,0 +1,24 @@
+        self.scaler = StandardScaler()
+        self.feature_columns = [
+            'entropy', 'correlation_h', 'correlation_v',
+            'noise', 'wavelet_total', 'contrast'
+        ]
+    
+    def prepare_features(self, metrics: Dict[str, float]) -> np.ndarray:
+        """Convert metrics dictionary to feature vector"""
+        features = [metrics[col] for col in self.feature_columns]
+        return np.array(features).reshape(1, -1)
+    
+    def predict_probability(self, metrics: Dict[str, float]) -> float:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)